discord/ext/tasks/__init__.py

```diff
@@ -276,6 +276,10 @@ def failed(self):
         """
         return self._has_failed
 
+    def is_running(self):
+        """:class:`bool`: Check if the task is currently running."""
+        return not self._task.done()
```

Comment:
This needs to handle the case where `self._task is None`

Reply:
Will do